src/dotnet/commands/dotnet-add/dotnet-add-proj/Program.cs

```diff
@@ -104,6 +104,20 @@ private void AddProject(SlnFile slnFile, string projectPath)
                 };
 
                 slnFile.Projects.Add(slnProject);
+
+                if (slnFile.ProjectConfigurationsSection.Any())
+                {
+                    var existingBuildConfigs = slnFile.ProjectConfigurationsSection.First();
```

Comment:
I'm assuming that the state of the sln file is correct before we add (otherwise I wouldn't have enough information to figure out what to add). So, I'm copying the build configurations for one of the potentially many projects specified in the solution.